### PR TITLE
Improve logging when creating and connecting to DB

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: tools/db_init.py [hunk 1]
```diff
@@ -132,3 +139,5 @@ def test_db(database):
            {test_cmd + db}
         '''))
     sys.exit(1)
+
+print()
```
Comment (profjsb) [inline tools/db_init.py:143]: what does this print statement do?

Comment (stefanv) [reply tools/db_init.py:143]: It was just for some spacing in the log so this is more clearly delineated. Not super important, feel free to remove.

**Comment by profjsb** [comment]
Is this ready to merge @stefanv ?

**Comment by stefanv** [comment]
Yes, please.

**Pull request merged**